craft_application/commands/lifecycle.py

```diff
@@ -77,6 +216,14 @@ def _fill_parser(self, parser: argparse.ArgumentParser) -> None:
             help="Build in a LXD container.",
         )
 
+        parser.add_argument(
+            "--pro",
+            type=ProServices.from_csv,
+            metavar="<pro-services>",
+            help="Enable Ubuntu Pro service by name. Each service should be separated by a comma.",
```

Comment:
Should we add that this only works if your base is Ubuntu-based? (e.g. charmcraft supports almalinux bases)

Reply:
That is a good point. Lets try suppressing the --pro help string, that way we will still raise an exception if the pro api is not found rather than a invalid command response. Is there a canonical method of detecting if we are running on a Ubuntu platform? Or should I check if `/usr/bin/ubuntu-advantage` exists

Reply:
We can't suppress the help string since at this point we won't know whether the actual build is going to run in an Ubuntu container or not.

Reply:
Sorry, I read that as running on Almalinux as directly, but that makes sense. I added a note to the help string. Thanks.